transform/app/services/s3/utils.py

```diff
@@ -1,8 +1,10 @@
+import gzip
+import json
 import logging
 import re
 import zipfile
 from io import BytesIO
-from typing import Generator, Optional
+from typing import Dict, Generator, List, Optional
```

Comment:
is there a need for Dict, List?

Reply:
We use typing and I wanted it to match that. But if it is not necessary, I change it.